libremesh/lime-app
# Dependencies last working ecosystem

This PR fixes dependencies version to those that were playing well together. So new developers can do git clone and  npm install without errors.

The most important ones are preact-cli, fixed to the last release that worked well in practice (3.0.0),
and storybook set of dependencies, fixed to the contemporary releases of the above. This is needed because they do not always specify all the dependencies in which they rely and end up using the ones installed by the other package.
So preact would end up using a package installed following storybook dependencies, or vice versa.

Also, some unused dependencies where removed.
Important dependencies versions are fixed in package.json (no ^<version> syntax).
Some minor changes due to dependencies updates were done.
package-lock.json is removed too, since it causes us more problems than solutions.

**How to test it?**

Clone, checkout, npm install in a clean directory, the run npm run test.
No errors (but many warnings) should be present.
